docs/src/routes/(docs)/components/button-ghost/+page.svelte

```diff
@@ -5,13 +5,23 @@
 <script>
   import { base } from "$app/paths";
   import Code from "$lib/Code.svelte";
+  import SideMenu from "$lib/SideMenu.svelte";
 </script>
 
 <svelte:head>
   <title>Ghost button</title>
 </svelte:head>
 
-<main id="main-content" tabindex="-1">
+<main class="sidemenu" id="main-content" tabindex="-1">
+  <SideMenu>
+    <ul>
+      <li><a href="#introduction">Introductie</a></li>
+      <li><a href="#examples">Voorbeelden</a></li>
+      <li><a href="#requirements">Bijbehorende bestanden</a></li>
```

Comment:
Wellicht wil je ook de #variables (instelbare variabelen) hier in het zijmenu hebben? 

Disclaimer: dit is vast op meer pagina's het geval. Ik heb ze niet allemaal gezien en/of van dit comment voorzien.

Reply:
we maken een aparte (nieuwe) PR met verdere aanpassingen aan de SideMenu's. Ik resolve deze comment